content/en/docs/admin/admission-controllers.md

```diff
@@ -60,13 +61,12 @@ admission control plugins:
 kube-apiserver --enable-admission-plugins=NamespaceLifecyle,LimitRanger ...
 ```
 
-{{< note >}}
 **Note**: Depending on the way your Kubernetes cluster is deployed and how the
 API server is started, you may need to apply the settings in different ways.
 For example, you may have to modify the systemd unit file if the API server is
 deployed as a systemd service, you may modify the manifest file for the API
 server if Kubernetes is deployed in a self-hosted way.
-{{< /note >}}
+{: .note}
```

Comment:
Use Hugo short-codes for this and all admonitions. See the source of https://raw.githubusercontent.com/kubernetes/website/master/content/en/docs/test.md for the syntax, or https://kubernetes.io/docs/test/ for what it looks like. Or just look at the red diff lines because it looks like this admonition was already updated to use the new syntax and your PR undoes it for some reason.